### DISCUSSION TITLE
Generating a Form does not adhere to the set 'class_namespace' set in the livewire config

### DISCUSSION THREAD

**Discussion by gijsbeijer** [livewire/livewire#7899] [7. Bugs (archived)]
### Livewire version

v3.3.4

### Laravel version

v10.43.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. publish the config\livewire.php
2. set the class_namespace to (e.g.) `"App\\View\\Livewire"`
3. generate a new form using: `php artisan livewire:form TestForm`
4. see the class created in `App\Livewire\Forms`

Expected:
See the class created in `App\View\Livewire\Forms`

### Repository or Wirebox URL

-

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@gijsbeijer thanks yeah that sounds like a bug. Any chance you could PR a fix? Thanks!

**Reply by gijsbeijer** [reply]
I can, but I won't be able to do this until somewhere next week or the week after due to personal circumstances. If I don't see a fix been made then, I will create a PR.

**Reply by joshhanley** [reply]
@gijsbeijer thanks! No rush, any help is much appreciated 🙂
